### DISCUSSION TITLE
How do I purge all information about backups?

### DISCUSSION THREAD

**Discussion by UntouchedWagons** [longhorn/longhorn#7742] [Q&A]
I recently rebuilt my cluster the other day for reasons unrelated to Longhorn and now in the Backups tab there's all these backups for the PVCs of the old cluster that I no longer need and I can't delete them. I've tried blanking the backup target which has sort of helped but there's still four ghose PVCs that won't go away.

![Firefox_Screenshot_2024-01-22T02-36-58 336Z](https://github.com/longhorn/longhorn/assets/160996/8331485a-a3cc-4479-9fec-5ba3740e7154)

If I click on the PVC name no backups are listed. If I put the backup target back and click save then go back to the Backups tab a whole bunch of ghost PVCs return and I get the warning "backup target URL is empty" which is demonstratively false.

How do I tell Longhorn to take whatever info it thinks it has on backups and just forget about it?

**Comment by c3y1huang** [comment]
Do those unwanted backups exist in the actual backup target, for example, the remote S3 bucket?

**Comment by UntouchedWagons** [comment]
I honestly don't remember. I cleared the backup target again, deleted the underlying dataset, remade the dataset and NFS share, waited about a day and Longhorn had finally deleted all that expired information. I decided to not use Longhorn's built-in backup system I can't depend on it.
